Sources/Basic/Path.swift

```diff
@@ -252,6 +258,12 @@ public struct RelativePath {
     public var suffix: String? {
         return _impl.suffix
     }
+
+    /// File extension of the give path's basename. This follow same rules as
+    /// suffix except that it doesn't include leading `.` character.
+    public var fileExtension: String? {
```

Comment:
I think we should probably call this `extension`, or call `suffix` `fileSuffix`. @abertelrud?